homework-2/routes/groupRouter.ts

```diff
@@ -0,0 +1,20 @@
+import { Router } from "express";
+import {
+    addGroupHandler,
+    addUserToGroup,
+    getAllGroupsHandler,
+    getGroupByIdHandler,
+    removeGroupHandler,
+    updateGroupHandler
+} from "../middewares/groupMiddlewares";
+
+const router: Router = Router();
+
+router.get('/add-user-to-group', addUserToGroup);
```

Comment:
You have to use post request for adding something

Reply:
/:id/users path would be better here

Reply:
Fixed, thanks